### PR TITLE
[CHORE] rewrite /@molgenis-ui to /ui

### PR DESCRIPTION
**Pull request by sidohaakma**
#### Checklist
- [x] Functionality works & meets specifications
- [x] Code reviewed
- [x] Code unit/integration/system tested
- [x] Migration step added in case of breaking change
- [x] User documentation updated
- [x] (If you have changed REST API interface) view-swagger.ftl updated
- [x] Test plan template updated
- [x] Clean commits
- [x] Added Feature/Fix to release notes

### CODE REVIEW

File: molgenis-core-ui/src/main/java/org/molgenis/core/ui/MolgenisWebAppConfig.java [hunk 1]
```diff
@@ -134,7 +134,7 @@ public void addResourceHandlers(ResourceHandlerRegistry registry) {
         .setCachePeriod(cachePeriod);
     registry
         .addResourceHandler(PATTERN_MOLGENIS_UI)
-        .addResourceLocations("/@molgenis-ui/", "classpath:/@molgenis-ui/")
+        .addResourceLocations("/ui/", "classpath:/@molgenis-ui/")
```
Comment (fdlk) [inline molgenis-core-ui/src/main/java/org/molgenis/core/ui/MolgenisWebAppConfig.java]: Looks like they need to match, this does not serve the resources

Comment (fdlk) [reply molgenis-core-ui/src/main/java/org/molgenis/core/ui/MolgenisWebAppConfig.java]: (at least I do not get a menu on the preview server)

File: molgenis-app/docker-compose.yml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 version: "3.1"
 services:
     nginx:
-      image: nginx:stable
+      image: molgenis/nginx:stable
```
Comment (connoratrug) [inline molgenis-app/docker-compose.yml]: is this a different image ? , or the same from other repo ?

Comment (sidohaakma) [reply molgenis-app/docker-compose.yml]: This is our own located on hub.docker.com on the molgenis organisation. The configuration is located in the molgenis-frontend repository. The version needs to be updated.

**Comment by dennishendriksen** [comment]
can be closed according to @sidohaakma. could you add a comment explaining why for future reference?

**Pull request closed without merging**